src/mailmojo-widget.php

```diff
@@ -61,8 +49,7 @@ public function initFiles () {
 	 * @param array $instance
 	 */
 	public function form ($instance) {
-		$mmApi = $this->mmPlugin->getApi();
-		if ($mmApi === null) {
+		if (empty($this->mmPlugin->username)) {
 			global $blog_id;
 			$adminUrl = get_admin_url($blog_id);
 			$output = "<p>%s <a href=\"{$adminUrl}options-general.php?page={$this->mmPlugin->getBasename()}\">%s</a>.</p>";
```

Comment:
Gah, GitHub.. Can't add comment on the lines below for some reason.. I noticed it in the translations files, and it's annoying that the text is split up into two gettext calls below. Seems like an good improvement to make while already updating the translations.